lib/Data/Chronicle/Reader.pm

```diff
@@ -105,9 +112,16 @@ sub get {
     my $name     = shift;
 
     my $key         = $category . '::' . $name;
-    my $cached_data = $self->cache_reader->get($key);
 
-    return JSON::from_json($cached_data) if defined $cached_data;
+    if ( defined $self->cache_reader ) {
+        my $cached_data = $self->cache_reader->get($key);
+        return JSON::from_json($cached_data) if defined $cached_data;
+    }
+
+    if ( defined $self->memory_map ) {
+        return $self->memory_map->{$key};
+    }
+
     return;
```

Comment:
I think we should fix this to `return undef;`. Returning an empty list for a method that's expecting to be called in scalar context is usually dangerous:

```perl
some_function($chronicle->get('missing key', 'data'), 1, 2, 3);
```

will call `some_function(1,2,3)`.